tshirts.c

```diff
@@ -3,19 +3,23 @@
 
 char size(int cms) {
     char sizeName = '\0';
-    if(cms < 38) {
+    if(cms <= 38) {
         sizeName = 'S';
     } else if(cms > 38 && cms < 42) {
         sizeName = 'M';
-    } else if(cms > 42) {
+    } else if(cms >= 42) {
         sizeName = 'L';
     }
     return sizeName;
 }
 
 int main() {
     assert(size(37) == 'S');
+    assert(size(38) == 'S');
+    assert(size(38) != 'M');
     assert(size(40) == 'M');
+    assert(size(42) != 'M');

```

Comment:
if size 42 is already == L , then how can it take any other value (!=M)? Isnt the condition redundant? unless you are trying to test the return of same value again and again..

Reply:
Yes, it seems redundant only. If it != M then it will be ==L only. Corrected t.